src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -24,7 +24,20 @@ export default class ContentCreatorComponent extends Component {
 
     handlePublish() {
         this._refIframe.contentWindow.onbeforeunload = () => {};
-        this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish').click();
+        this.submitForm();
+    }
+
+    submitForm() {
+        const selectors = ['#ezrepoforms_content_edit_publish', '#ezrepoforms_user_create_create'];
+
+        for(let selector of selectors) {
```

Comment:
What do you think about:
```js
const form = this._refIframe.contentWindow.document.body.querySelector('#ezrepoforms_content_edit_publish, #ezrepoforms_user_create_create');

form.click();
```
? Assuming that one of these forms exists _querySelector_ will return it.